pyk/Makefile

```diff
@@ -0,0 +1,57 @@
+.PHONY: all test integration-test check-code-style isort check-isort check-flake8 test-mypy
+
+all: check-code-style test
+
+
+# virtualenv
+
+VENV_DEV     := venv-dev/pyvenv.cfg
+ACTIVATE_DEV := . venv-dev/bin/activate
+
+$(VENV_DEV):
+	   virtualenv -p python3.8 venv-dev    \
+	&& $(ACTIVATE_DEV)                     \
+	&& pip install -r requirements/dev.txt
+
+
+VENV_PROD     := venv-prod/pyvenv.cfg
+ACTIVATE_PROD := . venv-prod/bin/activate
+
+$(VENV_PROD):
+	   virtualenv -p python3.8 venv-prod \
+	&& $(ACTIVATE_PROD)                  \
```

Comment:
Let's add .PHONY targets `venv-prod` and `venv-dev` so that we can easily set these up. Also do `@echo $(ACTIVATE_PROD)` at the end of the recipe, to print out what the user needs to do to directly activate the virtual environment themselves after doing `make venv-prod`. See https://github.com/runtimeverification/erc20-verification/blob/master/Makefile#L154

Reply:
Also, make `venv-dev` a variable, as `VENV_DEV_DIR := venv-dev`, and re-use that variable directly instead. Makes it easier to change that directory in the future.

Reply:
Done: 536a951188d26bf9c24f237500464740306287a1